docs/source/faqs.md

```diff
@@ -183,4 +183,39 @@ This has been solved in `ray>=2.47.1`, thus we could solve this as following:
 
 ```
 python3 -m pip install modelscope 'ray>=2.47.1' 'protobuf>3.20.0'
-``` 
+```
+
+### 21. Failed with inferencing Qwen3 MoE due to `Alloc sq cq fail` issue?
+
+When running Qwen3 MoE with tp/dp/ep, etc., you may encounter an error just as the following:
+
+```bash
```

Comment:
Please move detail log to a issue, and provide more info about version and branch info. And related PR.